util/expiresdict.py

```diff
@@ -50,7 +50,7 @@ def get(self, key, default_value=None):
         # Otherwise the key has expired or was not found. Rebuild the cache and check it again.
         items = self._rebuild()
         found_item = items.get(key)
-        if found_item is None:
+        if found_item is None or found_item.expired:
```

Comment:
Was this required for tests to pass?

Reply:
yes, otherwise the in-memory cache can incorrectly return an expired item